backend/src/main/java/com/soen341/instagram/exception/account/EmailTakenException.java

```diff
@@ -0,0 +1,13 @@
+package com.soen341.instagram.exception.account;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.BAD_REQUEST)
+public class EmailTakenException extends RuntimeException
+{
+	public EmailTakenException()
+	{
+		super("Email is already taken");
```

Comment:
Maybe something more descriptive like "This email already has an associated account"? This sounds like the user was trying to create an account (to me at least)

Reply:
Agreed